pkg/aspect/listCheckerManager.go

```diff
@@ -42,28 +42,37 @@ func NewListCheckerManager() Manager {
 }
 
 // NewAspect creates a listChecker aspect.
-func (listCheckerManager) NewAspect(cfg *CombinedConfig, ga adapter.Builder, env adapter.Env) (Wrapper, error) {
-	aa, ok := ga.(adapter.ListCheckerBuilder)
-	if !ok {
-		return nil, fmt.Errorf("adapter of incorrect type; expected adapter.ListCheckerBuilder got %#v %T", ga, ga)
+func (m listCheckerManager) NewAspect(c *CombinedConfig, b adapter.Builder, env adapter.Env) (Wrapper, error) {
+	aspectCfg := m.DefaultConfig().(*config.ListCheckerParams)
```

Comment:
aside: it sure feels like these structToProto() blocks should be in some way abstracted. other than the little bit of typing, these are just repeated from manager to manager.